coursefinder/views.py

```diff
@@ -1,31 +1,85 @@
 from django.shortcuts import render
 from django.http import HttpResponseRedirect
+from wagtail.core.models import Page
+from coursefinder.models import CourseFinderResults
+from django.conf import settings
 import requests
 
 # Create your views here.
 
 def results(request):
     course_query = request.GET.get('courseQuery', "")
     institution_query = request.GET.get('institutionQuery', "")
-    r = requests.get(url="http://host.docker.internal:10100/search/institution-courses?q=%s&institutions=%s" % (course_query, institution_query))
+    r = requests.get(url="%s/search/institution-courses?q=%s&institutions=%s" % (settings.SEARCHAPIHOST, course_query, institution_query))
     data = r.json()
     total_courses = data['total_number_of_courses']
     total_institutions = data['total_results']
     results = data['items']
+    page = CourseFinderResults.objects.get()
 
-    return render(request, 'coursefinder/results.html', {
+    return render(request, 'coursefinder/course_finder_results.html', {
+        'page': page,
         'results': results,
         'total_courses': total_courses,
         'total_institutions': total_institutions
     })
 
+
 def narrow_search(request):
     selection = request.POST.get('radioGroup', None)
+    course_query = request.POST.get('course_query', None)
+    mode_query = request.POST.get('mode_query', None)
+    countries_query = request.POST.get('countries_query', None)
+    print(mode_query)
+    url = "%s/search/institution-courses?subjects=%s" % (settings.SEARCHAPIHOST, course_query)
```

Comment:
It's not guaranteed that they will select a subject; so should this not be in an if statement?